### DISCUSSION TITLE
Split notes independent scroll bars, Basic Markdown editor

### DISCUSSION THREAD

**Discussion by JPL1** [standardnotes/forum#2910] [Suggestions]
Feature request.

I've just noticed the Edit | Split | Preview bar.

It's very nice, however Split could be improved by giving each side an independent scroll bar. That way it could be used like the split screen in MS Word, allowing you to work on a document with two different views. 

SN version = 3.101.2

**Comment by effieeee** [comment]
Hi @JPL1, I'm afraid the team may not prioritize fixes and requests on this since Basic Markdown has been deprecated, and is now a [community plugin](https://github.com/standardnotes/plugins) 😞
